analyzers/tests/SonarAnalyzer.TestFramework.Test/Common/SourceGeneratorProviderTest.cs

```diff
@@ -26,20 +26,20 @@ namespace SonarAnalyzer.TestFramework.Test.Common;
 public class SourceGeneratorProviderTest
 {
     private static AnalyzerFileReference RazorSourceGenerator =>
-        SourceGeneratorProvider.SourceGenerators.Single(x => x.FullPath.EndsWith("Microsoft.CodeAnalysis.Razor.Compiler.SourceGenerators.dll"));
+        SourceGeneratorProvider.SourceGenerators.Single(x => x.FullPath.EndsWith("Microsoft.CodeAnalysis.Razor.Compiler.dll"));
```

Comment:
The old library is no longer present. The source generators are now part of the `Microsoft.CodeAnalysis.Razor.Compiler.dll`.